iceoryx_hoofs/include/iceoryx_hoofs/internal/posix_wrapper/mutex.hpp

```diff
@@ -81,14 +81,15 @@ class mutex
     /// @brief  Tries to lock the mutex object. If it is not possible to lock
     ///         the mutex object try_lock will return an error. If the c
     ///         function fails it will return false, otherwise true.
-    bool try_lock();
+    bool tryLock();
```

Comment:
You did the right thing but here we explicitly followed the C++ STL code guidelines and for now I would stick to them.

The reason is when a user switches from C++ STL to our heapless, exceptionsless constructs that they have to adapt as less code as possible.

For this reason I also would stick to the lower case name `mutex`.